### PR TITLE
fix: Option to see Website

### LINKED ISSUE #1268: Add Option to visit website from app
**Describe the chore**
An option should be there to see the website.

**Would you like to work on the issue?**

- [x] Yes

### PR DESCRIPTION
**Pull request by somenath1435**
Fixes #1268 
Changes: Now website can be seen.
Screenshots:
Screenshot_20190311-072308
Screenshot_20190311-072301

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Updates? These are very small fixes. Why is taking so long?

**Comment by iamareebjamal** [comment]
You simply open a PR and then ignore any requested changes

**Comment by somenath1435** [comment]
@iamareebjamal I had already updated strings.xml

File: app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt [hunk 1]
```diff
@@ -54,6 +55,14 @@ class SettingsFragment : PreferenceFragmentCompat(), PreferenceChangeListener {
     }
 
     override fun onPreferenceTreeClick(preference: Preference?): Boolean {
+
+        if (preference?.key == resources.getString(R.string.visit_website)) {
+            // Goes to website
+            context?.let {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: No need for this. Use requireContext()

**Comment by ShridharGoel** [comment]
> @iamareebjamal I had already updated strings.xml

Where? You need to use separate string resources for key and title.

**Comment by somenath1435** [comment]
@iamareebjamal requested  changes has been made.

**Comment by iamareebjamal** [comment]
As a matter of fact, NONE of the requested changes have been made

**Comment by somenath1435** [comment]
@iamareebjamal I have used requireContext().
Can you please tell where to use it?

**Comment by iamareebjamal** [comment]
I must be blind then because I can't see. Neither requireContext nor the change requested by Shridhar Goel

https://github.com/fossasia/open-event-android/pull/1269/files

**Comment by somenath1435** [comment]
@iamareebjamal please refer to line 62

**Comment by iamareebjamal** [comment]
Please refer to the line I added my review to. It'll automatically get resolved once you remove context.let

**Comment by somenath1435** [comment]
Thanks for the reply

**Comment by somenath1435** [comment]
@iamareebjamal I have removed context.let

File: app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt [hunk 2]
```diff
@@ -54,6 +55,12 @@ class SettingsFragment : PreferenceFragmentCompat(), PreferenceChangeListener {
     }
 
     override fun onPreferenceTreeClick(preference: Preference?): Boolean {
+
+        if (preference?.key == resources.getString(R.string.visit_website)) {
+            // Goes to website
+                Utils.openUrl(requireContext(), WEBSITE_LINK)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: Wrong indentation

**Comment by somenath1435** [comment]
@iamareebjamal fixed indentation error

**Comment by somenath1435** [comment]
@iamareebjamal any other changes required?

**Pull request closed without merging**